### ISSUE TITLE
Result on MPI-INF-3DHP

### ISSUE THREAD

**Issue by CirceJie** [csiro-icvg/Diff3DHPE#3]
Hello! Using the provided checkpoint: ConditionalDiffusionMixSTES2SGRANDLinLift_l2_lr4e-4_useTembed_F_3dhp_gt_27f.bin without changing any parameters, the MPJPE value on the MPI-INF-3DHP dataset is 33, which is significantly different from the 19.6 reported in your paper. Could there be an error with the provided checkpoint?
I'm looking forward to your response.
![image](https://github.com/csiro-icvg/Diff3DHPE/assets/79498805/c8fb3f13-c66a-49bc-aae5-17c3caf368bb)

`Dataset: 3dhp,
Model: ConditionalDiffusionMixSTES2SGRANDLinLift

args=Namespace(actions='*', batch_size=64, batch_size_eval=512, beta_schedule='cosine', by_subject=False, checkpoint='checkpoint/3dhp/ConditionalDiffusionMixSTES2SGRANDLinLift/gt/f27/', checkpoint_frequency=40, clip_denoised=True, clip_loss=True, config='configs/3dhp_gt_s2s_ConditionalDiffusionMixSTES2SGRANDLinLift.json', data_augmentation=True, dataset='3dhp', ddim_sampling_eta=0.0, downsample=1, dropout=0.0, embed_dim=512, epochs=200, evaluate='ConditionalDiffusionMixSTES2SGRANDLinLift_l2_lr4e-4_useTembed_F_3dhp_gt_27f.bin', export_training_curves=False, gpu_id=['2,3,4,5'], keypoints='gt', learning_rate=4e-05, loss_type='mpjpe', lr_decay=0.99, max_time=48.0, model='ConditionalDiffusionMixSTES2SGRANDLinLift', no_eval=False, number_of_frames=27, out_all=True, patience=200, pretrained_model=None, repeat_n=None, resume='', sampling_timesteps=7, seed=42, stride=27, subjects_test='TS1,TS2,TS3,TS4,TS5,TS6', subjects_train='S1,S2,S3,S4,S5,S6,S7,S8', subjects_unlabeled='', subset=1, test_extra_noise_std=0.0, test_joint_drop=0.0, test_time_augmentation=True, timesteps=1000, train_extra_noise_std=0.0, train_joint_drop=0.0, train_val_extra_noise_std=0.0, train_val_joint_drop=0.0, transformer_depth=8, with_time_emb=False, workers=4)

Total Parameters: 33674755

'----'TS1'----'
Protocol #1 Error (MPJPE): 35.1246223449707mm
Protocol #2 Error (P-MPJPE): 12.493804931640625mm
Protocol #3 Error (N-MPJPE): 16.053863525390625mm
Velocity Error (MPJVE): 14.101750373840332mm
----------
'----'TS2'----'
Protocol #1 Error (MPJPE): 27.369386672973633mm
Protocol #2 Error (P-MPJPE): 17.400344848632812mm
Protocol #3 Error (N-MPJPE): 21.3829345703125mm
Velocity Error (MPJVE): 17.76490020751953mm
----------
'----'TS3'----'
Protocol #1 Error (MPJPE): 19.80672836303711mm
Protocol #2 Error (P-MPJPE): 13.26452922821045mm
Protocol #3 Error (N-MPJPE): 16.31395149230957mm
Velocity Error (MPJVE): 15.436038970947266mm
----------
'----'TS4'----'
Protocol #1 Error (MPJPE): 29.108449935913086mm
Protocol #2 Error (P-MPJPE): 14.961019515991211mm
Protocol #3 Error (N-MPJPE): 19.326440811157227mm
Velocity Error (MPJVE): 20.493589401245117mm
----------
'----'TS5'----'
Protocol #1 Error (MPJPE): 34.298282623291016mm
Protocol #2 Error (P-MPJPE): 21.439889907836914mm
Protocol #3 Error (N-MPJPE): 26.920289993286133mm
Velocity Error (MPJVE): 3.799125909805298mm
----------
'----'TS6'----'
Protocol #1 Error (MPJPE): 52.014122009277344mm
Protocol #2 Error (P-MPJPE): 27.27541732788086mm
Protocol #3 Error (N-MPJPE): 32.527992248535156mm
Velocity Error (MPJVE): 5.842132568359375mm
----------
Protocol #1   (MPJPE) action-wise average: 33.0mm
Protocol #2 (P-MPJPE) action-wise average: 17.8mm
Protocol #3 (N-MPJPE) action-wise average: 22.1mm
Velocity      (MPJVE) action-wise average: 12.91mm`

**Comment by socoolzjm** [comment]
Hi CirceJie, Did you use the processed data provided by https://github.com/paTRICK-swk/P-STMO? I ran the code and found no issue with reproduction. Please note that I found some inconsistent file names and corrected them, but those would not affect the result.
